CHANGELOG.md

```diff
@@ -4,11 +4,24 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## 0.1.2
+## [Unreleased]
+
+## [0.1.3] - 2018-12-11
+
+### Changed
+
+- Set max version to 10 because core platform is switching to Semver
+- Change debug message in settings page - [#43](https://github.com/owncloud/diagnostics/issues/43)
+
+## [0.1.2] - 2017-09-14
 
  - Fix deactivation of diagnostics app when occ is executed
 
-## 0.1.1
+## 0.1.1 - 2017-09-14
```

Comment:
Was this really the same release date as 0.1.2 ?

Reply:
yes it was, from here https://marketplace.owncloud.com/apps/diagnostics